pycbc/events/coinc_rate.py

```diff
@@ -62,6 +61,38 @@ def multiifo_noise_coinc_rate(rates, slop):
     return expected_coinc_rates
 
 
+def combination_noise_coinc_rate(rates, slop):
+    """
+    Calculate the expected rate of noise coincidences for a combination of
+    detectors
+
+    Parameters
+    ----------
+    rates: dict
+        Dictionary keyed on ifo string
+        Value is a sequence of single-detector trigger rates, units assumed
+        to be Hz
+    slop: float
+        time added to maximum time-of-flight between detectors to account
+        for timing error
+
+    Returns
+    -------
+    combo_coinc_rate: numpy array
+        Value is expected coincidence rate in the combination, units Hz
+    """
+    # extract ifo list and rates vectors from the dictionary
+    ifos = numpy.array(sorted(rates.keys()))
```

Comment:
i can't work out why this needs to be sorted. Presumably the calculation works OK if `ifos` were an unsorted array of ifo names?